### PR TITLE
Temp responsiveness

### PR DESCRIPTION
**Pull request by Kothalkarpriya**
added responsiveness to video library

### CODE REVIEW

**Review by rkapoor10 — APPROVED** [review]
The code looks fine. Breakpoints are well taken. You may look for screen values to adjust media queries. Rest everything is well structured. You can merge it.

File: src/assests/styles/video-card-page.css [hunk 1]
```diff
@@ -5,21 +5,85 @@ section {
...
+        padding: 5px;
+    }
+
+    .card-btn {
+        padding: 0;
+    }
+
+    .must-watch-video .video .image .img {
+        width: 210px;
+        height: 180px;
```
Comment (t007rushi) [inline src/assests/styles/video-card-page.css:41]: rem would be better choice here

File: src/assests/styles/video-card-page.css [hunk 2]
```diff
@@ -5,21 +5,85 @@ section {
...
+        margin: 0 1rem;
+    }
+
+    .video-card-page {
+        flex-direction: column;
+    }
+
+    .video-card-page aside .must-watch-video {
+        display: flex;
+        flex-direction: row;
```
Comment (t007rushi) [inline src/assests/styles/video-card-page.css:66]: ```row``` is by default direction

File: src/assests/styles/video-listing.css [hunk 3]
```diff
@@ -45,4 +46,31 @@
 
 .btn-icon{
     margin: 0 5px 0 0 ;
```
Comment (t007rushi) [inline src/assests/styles/video-listing.css]: it can be rewritten as margin-right : 5px;

**Review by t007rushi — APPROVED** [review]
just a suggestion to have light color be your cta color as light possible if its dark but seems less readable to user

File: src/assests/styles/video-card-page.css (hunk 2 above)
Comment (Kothalkarpriya) [reply src/assests/styles/video-card-page.css:66]: It was not working so I added the direction as row

File: src/assests/styles/video-listing.css (hunk 3 above)
Comment (Kothalkarpriya) [reply src/assests/styles/video-listing.css]: I will change

File: src/assests/styles/video-card-page.css (hunk 1 above)
Comment (Kothalkarpriya) [reply src/assests/styles/video-card-page.css:41]: yes

**Pull request merged**